src/actions/RowDetailActions.js

```diff
@@ -5,12 +5,14 @@
 
 export const ROW_DETAIL_KEYS = {
   ITEM_DETAIL: 'itemDetail',
-  REQUISITION_ITEM_DETAIL: 'requisitionItemDetail',
+  SUPPLIER_REQUISITION_ITEM_DETAIL: 'supplierRequisitionItemDetail',
+  CUSTOMER_REQUISITION_ITEM_DETAIL: 'customerRequisitionItemDetail',
 };
 
 export const ROW_DETAIL_ACTIONS = {
   OPEN_ITEM_DETAIL: 'rowDetail/openItemDetail',
-  OPEN_REQUISITION_ITEM_DETAIL: 'rowDetail/openRequisitionItemDetail',
+  OPEN_SUPPLIER_REQUISITION_ITEM_DETAIL: 'rowDetail/openSupplierRequisitionItemDetail',
+  OPEN_CUSTOMER_REQUISITION_ITEM_DETAIL: 'rowDetail/openCustomerRequisitionItemDetail',
```

Comment:
This `rowDetail` stuff is already implemented where it is quite easy to create a new component to show up in a bottom modal - these actions are essentially setting a `key` for the component to be displayed within the `rowDetail` component, which is actually in the root component